node/core/candidate-validation/src/lib.rs

```diff
@@ -859,15 +859,15 @@ mod tests {
 
 	#[test]
 	fn candidate_validation_bad_return_is_invalid() {
-		let validation_data: PersistedValidationData = Default::default();
+		let validation_data = PersistedValidationData::default().with(|vd| vd.max_pov_size=1024);
```

Comment:
```suggestion
		let validation_data = PersistedValidationdata { max_pov_size = 1024, ..Default::default());
```

For this you don't need any special function.